db/migrate/20200825080245_create_generic_events.rb

```diff
@@ -0,0 +1,18 @@
+class CreateGenericEvents < ActiveRecord::Migration[6.0]
+  def change
+    create_table :generic_events, id: :uuid do |t|
+      t.uuid :eventable_id, null: false     # polymorphic eventable/subject of the event
```

Comment:
are we always sure we will have an associated subjects per event, can an event not be associated to anything for more general events?

Reply:
Not currently, no. It'll be about a Person if it's more generic.